qutebrowser/browser/webkit/webpage.py

```diff
@@ -186,6 +186,18 @@ def _handle_errorpage(self, info, errpage):
             errpage.encoding = 'utf-8'
             return True
 
+    def chooseFile(self, parent_frame: QWebFrame, suggested_file: str):
+        """Override chooseFiles to (optionally) invoke custom file uploader."""
+        handler = config.val.fileselect.handler
+        if handler == "default":
+            return super().chooseFile(parent_frame, suggested_file)
+
+        selected_file = shared.choose_file(multiple=False)
+        if selected_file == []:
```

Comment:
Tiny nit: I would prefer `not selected_file` or `len(selected_file) == 0`, as if choose_file starts returning, for example, tuples instead of lists, this won't break.

Reply:
+1 for `not selected_file`

Reply:
The `not` options is faster as well, so it'd be a good idea to use that.

Reply:
@user202729 please check it out again because it is still ```if selected_file == []:```.